agent/cita_exporter/cita_monitor_agent.py

```diff
@@ -266,6 +272,14 @@ def exporter():
         return Response(prometheus_client.generate_latest(registry), mimetype="text/plain")
 
     service_status.labels(NodeIP=node_ip, NodePort=node_port).set(1)
+
+    network_port = int(os.getenv('NETWORK_PORT'))
```

Comment:
can't we get this via config? we already have this `SOFT_PATH`

Reply:
the port write in config file is container port, may not the port mapping to the host. i just follow the way of json rpc port